staker/reward_recipient_store.gno

```diff
@@ -195,6 +195,33 @@ func NewPoolLiquidity() *PoolLiquidity {
 	}
 }
 
+// AddInRangePosition adds a new in-range position to the pool
+func (p *PoolLiquidity) AddInRangePosition(tokenID uint64, position *InRangeLiquidity) {
+	// nil check for map
+	if p.inRangeLiquidityMap == nil {
+		p.inRangeLiquidityMap = make(map[uint64]*InRangeLiquidity)
+	}
+
+	// nil check for position
+	if position == nil {
+		position = NewInRangeLiquidity()
+	}
+
+	// nil check for totalLiquidity
+	if p.totalLiquidity == nil {
+		p.totalLiquidity = u256.Zero()
+	}
+
+	// Add position's liquidity to total liquidity
+	positionLiquidity := position.GetLiquidity()
+	if positionLiquidity != nil {
+		p.totalLiquidity.Add(p.totalLiquidity, positionLiquidity)
```

Comment:
liquidity is already huge number (that's why we use uint256 for it).
adding u256 + u256 does need overflow check.

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/437/commits/9a1d977976448874c848a1a4d1a2ff63a03357a3